pallets/storage-provider/src/tests/mod.rs

```diff
@@ -107,11 +108,14 @@ impl ProofVerification for DummyProofsVerification {
         _replicas: BoundedBTreeMap<
             SectorNumber,
             PublicReplicaInfo,
-            ConstU32<MAX_SECTORS_PER_PROOF>,
+            ConstU32<{ MAX_SECTORS_PER_PROOF * MAX_PROOFS_PER_BLOCK }>,
+        >,
+        proofs: BoundedVec<
+            BoundedVec<u8, ConstU32<MAX_POST_PROOF_BYTES>>,
+            ConstU32<MAX_PROOFS_PER_BLOCK>,
         >,
-        proof: BoundedVec<u8, ConstU32<MAX_POST_PROOF_BYTES>>,
     ) -> sp_runtime::DispatchResult {
-        if *proof == INVALID_PROOF {
+        if *proofs[0] == INVALID_PROOF {
             return Err(sp_runtime::DispatchError::Other("invalid proof"));
```

Comment:
Does this still match the docs for `DummyProofsVerification`?

Reply:
good catch, I'll update the docs.